rotkehlchen/rotkehlchen.py

```diff
@@ -558,7 +558,16 @@ def query_balances(
 
         result_dict = merge_dicts(combined, stats)
 
-        allowed_to_save = requested_save_data and self.data.should_save_balances()
+        """
```

Comment:
Don't write a docstring like that for a comment unless it's in the start of the function. Use `#` instead. But here you don't need to over-explain it. Good variable names have the same effect. Example:

```python
allowed_to_save = requested_save_data or self.data.should_save_balances()
```

This would result in saving either if we specifically request it, or if the time has come.